doc/source/getting_started/write_doc/tutorial_card_template.rst

```diff
@@ -0,0 +1,14 @@
+.. grid:: 1 1 3 3
+    :gutter: 2
+    :padding: 2
+    :margin: 2
+
+    .. grid-item-card:: Tutorial title
+       :link: ref
+       :link-type: ref
+       :text-align: center
+
+       This tutorial ...
+
+       +++
+       :bdg-mapdl:`MAPDL` :bdg-lsdyna:`LS-DYNA` :bdg-fluent:`FLUENT` :bdg-cfx:`CFX`
```

Comment:
```suggestion
.. grid-item-card:: Tutorial title
   :link: ref
   :link-type: ref
   :text-align: center

   This tutorial ...

   +++
   :bdg-mapdl:`MAPDL` :bdg-lsdyna:`LS-DYNA` :bdg-fluent:`FLUENT` :bdg-cfx:`CFX`
```
I think you can remove the grid command from the template as this is to show the card in particular

Reply:
And you can then include it directly in the guidelines with `literalinclude`.